hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OzoneManagerUtils.java

```diff
@@ -62,7 +62,7 @@ private OzoneManagerUtils() {
    * omMetadataManager().getBucketTable().get(buckKey)
    */
 
-  private static OmBucketInfo getOmBucketInfo(OMMetadataManager metaMgr,
+  public static OmBucketInfo getOmBucketInfo(OMMetadataManager metaMgr,
```

Comment:
It's used for legacy buckets. This PR it's just the existing code that has been refactored. We must have missed it, I will change it back. It will make sense in the next PR where it will be used by the legacy bucket code.

Reply:
Got it. In the case would you also clarify this in the PR title (and jira) a bit more?

e.g.: `HDDS-5504. Refactor namespace summaries (du, dist & counts) for upcoming legacy FS buckets support`